frontend/packages/noobaa-storage-plugin/src/components/status-card/statuses.tsx

```diff
@@ -1,77 +1,51 @@
 import * as _ from 'lodash';
+import { PrometheusHealthHandler } from '@console/plugin-sdk';
 import { HealthState } from '@console/shared/src/components/dashboard/status-card/states';
-import { PrometheusResponse } from '@console/internal/components/graphs';
-import { FirehoseResult } from '@console/internal/components/utils';
 import { getGaugeValue } from '../../utils';
 
-const NooBaaStatus = [
-  {
-    state: HealthState.ERROR,
-    message: 'MCG is not running',
-  },
-  {
+const NooBaaStatus = {
+  '0': { state: HealthState.OK },
+  '1': {
     state: HealthState.ERROR,
     message: 'All resources are unhealthy',
   },
-  {
+  '2': {
     state: HealthState.WARNING,
     message: 'Object Bucket has an issue',
   },
-  {
+  '3': {
     state: HealthState.ERROR,
     message: 'Many buckets have issues',
   },
-  {
+  '4': {
     state: HealthState.WARNING,
     message: 'Some buckets have issues',
   },
```

Comment:
```suggestion
  },
  '5': { 
          state: HealthState.UNKNOWN 
       }
       
```

Reply:
The set of status , we are receiving are { 0,1,2,3,4 }.
I intentionally omit '5' as this is not  a valid status key.

Reply:
Correct but can be used as default. If not 5 we can use X number as we also follow same pattern in ceph also.

Reply:
In ceph, we are not using that now, we have a map of statuses, there too.